Client/trunk/ParaEngineClient/Engine/ParaEngineApp.cpp

```diff
@@ -2067,6 +2067,7 @@ HRESULT CParaEngineApp::DoWork()
 
 HRESULT CParaEngineApp::Render3DEnvironment(bool bForceRender)
 {
+	++m_nCurrentFrameNumber;
```

Comment:
There is already one in `void SetFrameNumber(int nFrameNumber);`  in SceneObject. 
It works when there are multiple viewports.